### PR TITLE
Add english readme

### PR DESCRIPTION
**Pull request by rainyfly**
#目标检测
yolov3_darknet53_pedestrian
ssd_mobilenet_v1_pascal
yolov3_resnet34_coco2017
ssd_vgg16_512_coco2017
yolov3_resnet50_vd_coco2017
yolov3_darknet53_vehicles
yolov3_darknet53_coco2017
faster_rcnn_resnet50_fpn_venus
yolov3_mobilenet_v1_coco2017
faster_rcnn_resnet50_fpn_coco2017
yolov3_darknet53_venus
faster_rcnn_resnet50_coco2017
#人脸检测
pyramidbox_lite_server
pyramidbox_lite_server_mask
ultra_light_fast_generic_face_detector_1mb_640
pyramidbox_lite_mobile
pyramidbox_lite_mobile_mask
ultra_light_fast_generic_face_detector_1mb_320
pyramidbox_face_detection
#深度估计
MiDaS_Large
#图像生成
animegan_v2_hayao_99
animegan_v2_shinkai_53
animegan_v2_paprika_98
stylepro_artistic
animegan_v2_shinkai_33
animegan_v1_hayao_60
animegan_v2_hayao_64
U2Net_Portrait
Photo2Cartoon
stgan_bald
UGATIT_100w
animegan_v2_paprika_74
#文字识别
japan_ocr_db_crnn_mobile
german_ocr_db_crnn_mobile
Vehicle_License_Plate_Recognition
#图像分类
vgg19_imagenet
xception71_imagenet
xception41_imagenet
se_resnext50_32x4d_imagenet
resnext50_vd_32x4d_imagenet
alexnet_imagenet
mobilenet_v2_imagenet
se_resnext101_32x4d_imagenet
shufflenet_v2_imagenet
mobilenet_v3_large_imagenet_ssld
DriverStatusRecognition
mobilenet_v2_animals
vgg16_imagenet
vgg13_imagenet
efficientnetb7_imagenet
resnet_v2_101_imagenet
resnet_v2_18_imagenet
densenet161_imagenet
googlenet_imagenet
darknet53_imagenet
resnet_v2_152_imagenet
resnext101_64x4d_imagenet
resnext101_vd_64x4d_imagenet
mobilenet_v3_small_imagenet_ssld
resnet50_vd_dishes
nasnet_imagenet
dpn107_imagenet
inception_v4_imagenet
resnext50_32x4d_imagenet
resnext152_64x4d_imagenet
resnext50_64x4d_imagenet
densenet169_imagenet
resnet50_vd_wildanimals
efficientnetb4_imagenet
efficientnetb0_small_imagenet
resnext101_32x4d_imagenet
SnakeIdentification
res2net101_vd_26w_4s_imagenet
se_resnet18_vd_imagenet
mobilenet_v2_dishes
efficientnetb6_imagenet
resnet18_vd_imagenet
resnext50_vd_64x4d_imagenet
resnext101_32x16d_wsl
pnasnet_imagenet
vgg11_imagenet
fix_resnext101_32x48d_wsl_imagenet
marine_biometrics
resnext101_vd_32x4d_imagenet
spinalnet_res101_gemstone
resnext152_vd_64x4d_imagenet
mobilenet_v2_imagenet_ssld
xception65_imagenet
resnet50_vd_10w
efficientnetb5_imagenet
efficientnetb2_imagenet
efficientnetb1_imagenet
dpn92_imagenet
densenet121_imagenet
resnext152_32x4d_imagenet
resnet_v2_34_imagenet
resnext101_32x48d_wsl
dpn68_imagenet
spinalnet_vgg16_gemstone
food_classification
resnext101_32x32d_wsl
densenet201_imagenet
efficientnetb0_imagenet
resnext101_32x8d_wsl
efficientnetb3_imagenet
dpn131_imagenet
densenet264_imagenet
spinalnet_res50_gemstone
dpn98_imagenet
resnet_v2_50_imagenet
#实例分割
FCN_HRNet_W18_Face_Seg

### CODE REVIEW

File: modules/image/Image_gan/gan/stgan_bald/README_en.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# stgan_bald
+
+|Module Name|stgan_bald|
+| :--- | :---: |
+|Category|image generation|
+|Network|STGAN|
+|Dataset|CelebA|
+|Fine-tuning supported or not|No|
+|Module Size|287MB|
```
Comment (linjieccc) [inline modules/image/Image_gan/gan/stgan_bald/README_en.md:9]: Module Size -> Module size非首字母可以和其他行对齐

File: modules/image/Image_gan/gan/stgan_bald/README_en.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      - paths (list[str]): image path;
+      - use_gpu (bool): use GPU or not; **set the CUDA_VISIBLE_DEVICES environment variable first if you are using GPU**
+      - visualization (bool): Whether to save the results as picture files;
+      - output_dir (str): save path of images;
+
+      **NOTE:** choose one parameter to provide data from paths and images
+
+    - **Return**
+
+      - res (list\[numpy.ndarray\]): result list，ndarray.shape 为 \[H, W, C\]
```
Comment (linjieccc) [inline modules/image/Image_gan/gan/stgan_bald/README_en.md]: shape is

File: modules/image/Image_gan/style_transfer/U2Net_Portrait/README.md [hunk 3]
```diff
@@ -1,103 +1,103 @@
...
-## 四、更新历史
-
-* 1.0.0
-
-  初始发布
-
-  - ```shell
-    $ hub install U2Net_Portrait==1.0.0
-    ```
+# U2Net_Portrait
```
Comment (linjieccc) [inline modules/image/Image_gan/style_transfer/U2Net_Portrait/README.md:1]: 这个仍然是中文文档

File: modules/image/Image_gan/style_transfer/UGATIT_100w/README.md [hunk 4]
```diff
@@ -50,7 +50,7 @@
 
 ## 三、模型API预测
```
Comment (linjieccc) [inline modules/image/Image_gan/style_transfer/UGATIT_100w/README.md:51]: 替换为英文文档

File: modules/image/Image_gan/style_transfer/animegan_v1_hayao_60/README.md [hunk 5]
```diff
@@ -51,7 +51,7 @@
 
 ## 三、模型API预测
```
Comment (linjieccc) [inline modules/image/Image_gan/style_transfer/animegan_v1_hayao_60/README.md:53]: 同上

**Review by linjieccc — COMMENTED** [review]
需要再整体检查一下，一些还是中文文档

File: modules/image/Image_gan/gan/stgan_bald/README_en.md (hunk 1 above)
Comment (rainyfly) [reply modules/image/Image_gan/gan/stgan_bald/README_en.md:9]: 好像大家的文档都是这样的，应该是模板就是这样的

File: modules/image/Image_gan/gan/stgan_bald/README_en.md (hunk 2 above)
Comment (rainyfly) [reply modules/image/Image_gan/gan/stgan_bald/README_en.md]: 已改

File: modules/image/Image_gan/style_transfer/U2Net_Portrait/README.md (hunk 3 above)
Comment (rainyfly) [reply modules/image/Image_gan/style_transfer/U2Net_Portrait/README.md:1]: 有一些是中文文档，因为之前有的中文文档由于没有把"示例代码"改成"预测示例代码", 所以将没改的统一给改了。

**Comment by rainyfly** [comment]
已全部自查

**Review by KPatr1ck — APPROVED** [review]

**Pull request merged**